docs/src/content/docs/tr/guides/overriding-components.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Bileşenleri Geçersiz Kılma
+description: Starlight'ın yerleşik bileşenlerini geçersiz kılma ve belgelerinizin kullanıcı arayüzüne özel öğeler eklemek için nasıl yapılacağını öğrenin.
+sidebar:
+  badge: New
+---
+
+Starlight'ın varsayılan kullanıcı arayüzü ve yapılandırma seçenekleri, çeşitli içerikler için esnek olacak şekilde tasarlanmıştır. Starlight'ın varsayılan görünümünün büyük bir kısmı özel [CSS](/guides/css-and-tailwind/) ve [yapılandırma seçenekleri](/guides/customization/) ile özelleştirilebilir.
+
+Kutudan çıkabileceklerin ötesinde daha fazlasına ihtiyacınız olduğunda, Starlight, varsayılan bileşenlerini genişletmek veya geçersiz kılmak (tamamen değiştirmek) için kendi özel bileşenlerinizi oluşturmanızı destekler.
+
+## Ne Zaman Geçersiz Kılmak Gereklidir
+
+Starlight'ın varsayılan bileşenlerini geçersiz kılmak, aşağıdaki durumlarda faydalı olabilir:
+
+- Starlight'ın kullanıcı arayüzünün bir bölümünün görünümünü [özel CSS](/guides/css-and-tailwind/) ile değiştirmek mümkün olmadığında.
+- Starlight'ın kullanıcı arayüzünün bir bölümünün davranışını değiştirmek istediğinizde.
+- Starlight'ın mevcut kullanıcı arayüzüne ek olarak bazı ek kullanıcı arayüzü eklemek istediğinizde.
+
+## Geçersiz Kılma Nasıl Yapılır
+
+1. Geçersiz kılmak istediğiniz Starlight bileşenini seçin.
+   Tüm bileşenlerin tam listesini [Geçersiz Kılma Referansı](/reference/overrides/) sayfasında bulabilirsiniz.
+
+   Bu örnek, sayfa gezinme çubuğundaki Starlight'ın [`SocialIcons`](/reference/overrides/#socialicons) bileşenini geçersiz kılacak.
+
+2. Starlight bileşenini değiştirmek için kullanılacak bir Astro bileşeni oluşturun.
+   Bu örnek bir iletişim bağlantısı oluşturur. 
+
+   ```astro
+  ---
+  // src/components/EmailLink.astro
+  import type { Props } from '@astrojs/starlight/props';
+  ---
+
+  <a href="mailto:houston@example.com">Bana E-posta Gönder</a>
```

Comment:
Looks like the indentation here is causing issues:

<img width="795" alt="image" src="https://github.com/withastro/starlight/assets/357379/330fcc81-37c3-4d80-8f22-db7fcc920d2b">


This should fix it:

```suggestion
   ---
   // src/components/EmailLink.astro
   import type { Props } from '@astrojs/starlight/props';
   ---

   <a href="mailto:houston@example.com">Bana E-posta Gönder</a>
```